### ISSUE TITLE
TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by phet-steele** [phetsims/masses-and-springs#67]
```
TypeError: Cannot read property 'get' of undefined
    at https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Multilink.js?bust=1491294341966:29:84
    at Array.map (native)
    at new Multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Multilink.js?bust=1491294341966:29:42)
    at Function.multilink (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Property.js?bust=1491294341966:457:16)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/masses-and-springs/js/common/view/MassNode.js?bust=1491294341966:135:18)
    at Emitter.emit2 (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Emitter.js?bust=1491294341966:147:49)
    at Property._notifyObservers (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Property.js?bust=1491294341966:208:29)
    at Property._setAndNotifyObservers (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Property.js?bust=1491294341966:197:14)
    at Property.set (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/axon/js/Property.js?bust=1491294341966:141:16)
    at Spring.setMass (https://bayes.colorado.edu/continuous-testing/snapshot-1491286975754/masses-and-springs/js/common/model/Spring.js?bust=1491294341966:265:46)
Approximately 4/4/2017, 12:22:55 AM
```

**Comment by Denz1994** [comment]
Multilink should be moved to model for Vector screen. The current implementation has it in the common model. This won't be a quick fix and will require significant refactoring of the multiLink.

Thanks for the report.

**Comment by Denz1994** [comment]
I believe this was fixed in commits  8f8097809ee0b8e389757c293d5e07680f01f5f0 and 3b7f72e9465fc2667f1ac34cdafc98c16b66d96e. I cannot reproduce this error as well.
